examples/password-manager/src/3_vault.tsx

```diff
@@ -36,17 +50,13 @@ const VaultPage: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const items = me.root?.folders?.flatMap(
+  const items = me?.root.folders.flatMap(
     (folder) =>
       folder?.items?.filter(
         (item): item is Exclude<typeof item, null> => !!item,
       ) || [],
   );
-  const folders = useCoState(co.list(Folder), me.root?._refs.folders?.id, {
```

Comment:
Found that passing the schema this way to `useCoState` creates infinte loops in React 19.